Work with names/Work with names/Program.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+
+namespace Work_with_names
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            {
+                Console.WriteLine("Обрабатываю данные");
+                string path = "names.txt";
+
+                using (StreamReader reader = new StreamReader(path, Encoding.UTF8))
+                {
+                    List<string> names = new List<string>();
+                    while (!reader.EndOfStream)
+                    {
+                        string name = reader.ReadLine();
+                        names.Add(name);
+                    }
```

Comment:
Можно было сразу создавать список уникальных имен, чтобы второй раз не проходить по всепму списку и занимать меньше памяти.
```suggestion
                    while (!reader.EndOfStream)
                    {
                        string name = reader.ReadLine();
                        name = char.ToUpper(name[0]) + name.Substring(1);
                        
                        if(!names.Contains(name))
                        {
                            names.Add(name);
                            Console.Write(".");
                        }
                    }
```